src/main/kotlin/io/bazel/kotlin/builder/toolchain/KotlinToolchain.kt

```diff
@@ -104,6 +104,27 @@ class KotlinToolchain private constructor(
         ).toPath()
     }
 
+    private val KOTLINX_SERIALIZATION_CORE_JVM by lazy {
```

Comment:
The keep decomposing the std libs....

Reply:
this was a new dep added on ksp's symbol-processing